modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -251,6 +256,8 @@ public class PartitionReplicaLifecycleManager extends
      * @param sharedTxStateStorage Shared tx state storage.
      * @param txManager Transaction manager.
      * @param schemaManager Schema manager.
+     * @param dataStorageManager Data storage manager for checking zones' storage profiles for volatile flag.
```

Comment:
Let's not mention why we need it. This might easily be changed, and the javadoc will not be updated, so it will become stale

Reply:
Mostly I'd like to read a motivation under one-time added dependencies to be sure when I want to delete them that there no any other reasons to keep them. But it's opinionated.

Reply:
When a dependency is not used anymore, the IDE will highlight them as unused; maybe PMD/checkstyle will fail, so in most cases a comment is not even needed

Reply:
I removed the tail of the comment.